src/persist-client/Cargo.toml

```diff
@@ -44,6 +44,7 @@ mz-persist = { path = "../persist" }
 mz-persist-types = { path = "../persist-types" }
 mz-proto = { path = "../proto" }
 mz-timely-util = { path = "../timely-util" }
+paste = "1.0.11"
```

Comment:
is this from a previous version of the pr?

Reply:
no... I needed to import it for the macro to work. is there a way around this?

Reply:
yes, you can do this in the macro crate:
```
#[doc(hidden)]
pub use paste;
```

and have the macro use `$crate::paste` (`$crate` is special and refers to the macro crate)